.github/workflows/issues.yml

```diff
@@ -0,0 +1,65 @@
+---
+  name: Create JIRA ticket for new issues
+  
+  on:
+    issues:
+      types: [opened]
+  permissions: 
+    issues: write
+    contents: read
+  jobs:
+    jira_task:
+      name: Create Jira issue
+      runs-on: ubuntu-latest
+      steps:
+      - name: Create JIRA ticket
+        id: create
+        shell: bash
+        env:
+          ISSUE_TITLE: ${{ github.event.issue.title }}
+          ISSUE_BODY: ${{ github.event.issue.body }}
+          ISSUE_URL: ${{ github.event.issue.html_url }}
+          JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+          JIRA_ASSIGNEE: ${{ secrets.ASSIGNEE_JIRA_TICKET }}
+        run: |
+
+          # Use the tr command to remove the special characters from ISSUE_TITLE
+          ISSUE_TITLE_NO_SPECIAL_CHARS=$(echo "$ISSUE_TITLE" | tr -d '!$&*@#"\047\140')
+          json_response=$(curl --request POST \
+            --url 'https://jira.mongodb.org/rest/api/2/issue' \
+            --header 'Authorization: Bearer '"${JIRA_API_TOKEN}" \
+            --header 'Accept: application/json' \
+            --header 'Content-Type: application/json' \
+            --data '{
+              "fields": {
+                  "project": {
+                      "id": "19383"
+                  },
+                  "summary": "HELP: '"${ISSUE_TITLE_NO_SPECIAL_CHARS}"'",
+                  "issuetype": {
+                      "id": "3"
+                  },
+                  "description": "This ticket tracks the following GitHub issue: '"${ISSUE_URL}"'.",
+                  "components": [
+                      {
+                          "id": "27590"
```

Comment:
make sure the component id is the one for CLOUDP

Reply:
I shamelessly copied your work from Terraform without attribution. 
We need to figure out secrets we need for that job